cpu/stm32/kconfigs/Kconfig.clk

```
@@ -147,7 +259,7 @@ config CLOCK_APB1_DIV
     default 16 if CLOCK_APB1_DIV_16
 
 choice
-bool "APB2 prescaler (division factor of HCLK to produce PCLK2)"  if CPU_FAM_G4
+bool "APB2 prescaler (division factor of HCLK to produce PCLK2)"  if CPU_FAM_G4 || CPU_FAM_L0 || CPU_FAM_L1
```

Maybe here we can also move it to a new line?
```suggestion
bool "APB2 prescaler (division factor of HCLK to produce PCLK2)"
depends on CPU_FAM_G4 || CPU_FAM_L0 || CPU_FAM_L1
```